src/test/java/easytests/support/IssueStandardSupport.java

```diff
@@ -0,0 +1,153 @@
+package easytests.support;
+
+import easytests.core.entities.IssueStandardEntity;
+import easytests.core.models.IssueStandardModelInterface;
+import easytests.core.models.empty.ModelsListEmpty;
+import easytests.core.models.empty.SubjectModelEmpty;
+import org.junit.Assert;
+import org.mockito.Mockito;
+
+public class IssueStandardSupport {
+    protected static Object[][] fixtures = new Object[][]{
+            {
+                    1,
+                    300,
+                    30,
+                    1
+            },
+            {
+                    2,
+                    null,
+                    15,
+                    3
+            },
+            {
+                    5,
+                    3600,
+                    30,
+                    3
+            }
+    };
+
+    protected static Object[][] additional = new Object[][]{
+            {
+                    // for insert entity
+                    null,
+                    3600,
+                    20,
+                    2
+            },
+            {
+                    // for update entity with id = 1
+                    1,
+                    3600,
+                    20,
+                    2
+            },
+    };
+
+    public IssueStandardEntity getEntityFixtureMock(Integer index){
```

Comment:
in all methods there must be space after close bracket  `) {`
Fix all cases in support method. 

NB! In array definition `){` is OK